site/templates/page/page-header.jet

```diff
@@ -1,13 +1,13 @@
 {{block pageHeader(title="", plan=0)}}
   {{if len(title) > 0}}
-    <div class="page-header">
+    <div class="page-header pb-0">
       {{if plan != 0}}
         <s72-plan-label data-slug="{{plan.Slug}}" data-title="{{plan.Name}}"></s72-plan-label>
       {{end}}
       <h1>{{title}}</h1>
-      {{if plan != 0 && plan.HasExpiryDate() }}
-        {{ if plan.HasExpiryDate() }}
-          <p class="alert p-0 border-0 my-1">
```

Comment:
Re-add back in the 'alert' inline class and good to go (without it, the icon color rule [here](https://github.com/shift72/core-template/blob/develop/site/styles/_icons.scss#L35-L37) is not primary as per [design](https://app.zeplin.io/project/607f961cf9f03790cc66e475/screen/61396196750e52b971758c75))